src/server.js

```diff
@@ -202,6 +210,48 @@ app.get(
   }
 );
 
+app.get(
+  `${PREFIX_API}/:${URL_PARAM_EVENT_ID}/restaurants`,
+  getEvent,
+  async (request, response) => {
+    const { event } = request;
+    const users = event.users || {};
+    const userInput = Object.values(users);
+
+    if (userInput.length > 0) {
```

Comment:
Do you expect userInput.length to ever be > 1? If not, it could more helpful to check that userInput.length == 1, as this would reveal bugs.

Reply:
So here, I check for if userInput is empty. If true, there is no user Location to make the API call with. Does this answer your question? Do you suggest I check if userInput == 0 first?